contribs/gmf/src/print/component.js

```diff
@@ -628,8 +628,9 @@ export class PrintController {
       getRotationFn = () => this.rotation;
     }
 
-    this.postcomposeListener_ = this.ngeoPrintUtils_.createPrintMaskPostcompose(
-      getSizeFn, this.getScaleFn.bind(this), getRotationFn);
```

Comment:
Shouldn't we remove createPrintMaskPostcompose?

Reply:
it's still used in one example, I will change it

Reply:
done

Reply:
ok, thanks :-)